x-pack/plugin/security/qa/jwt-realm/build.gradle

```diff
@@ -9,6 +10,8 @@ dependencies {
   javaRestTestImplementation project(":client:rest")
 }
 
+boolean explicitIdTokenType = (new Random(Long.parseUnsignedLong(BuildParams.testSeed.tokenize(':').get(0), 16))).nextBoolean()
```

Comment:
Do we really need that much boiler plate to randomise something in gradle? Maybe we should talk about adding a couple of utility methods into `org.elasticsearch.gradle.` somewhere.
Even just a `getTestRandom()` would help, but `randomBoolean()` and `randomFrom(...)` would probably be justifed.

Reply:
Good point. I raised https://github.com/elastic/elasticsearch/pull/91674